src/resources/views/character/sheet.blade.php

```diff
@@ -198,9 +198,9 @@
                   <dt>{{ trans('web::seat.bonus_remaps') }}</dt>
                   <dd>{{ $character->pilot_attributes->bonus_remaps ?: 0 }}</dd>
                   <dt>{{ trans('web::seat.last_remap_date') }}</dt>
-                  <dd>{{ $character->pilot_attributes->last_remap_date ?: carbon() }}</dd>
+                  <dd>{{ $character->pilot_attributes->last_remap_date ?: trans('web::seat.no_remap') }}</dd>
                   <dt>{{ trans('web::seat.accrued_remap_cooldown_date') }}</dt>
-                  <dd>{{ $character->pilot_attributes->accrued_remap_cooldown_date ?: carbon() }}</dd>
```

Comment:
isn't that still valid if there is no remap ? I think it was the "next available remap date"